frontend/src/app/components/overview/overview.component.ts

```diff
@@ -18,83 +15,63 @@ import {Area} from '../../model/area';
 export class OverviewComponent implements OnInit {
 
   positiveCasesPerDateData: any;
-  sexDistributionCasesData: SexDistribution;
-  sexDistributionDeathsData: SexDistribution;
-  hospitalBedsPerDate: HospitalBedsDaily;
+  sexDistributionCasesData: any;
+  sexDistributionDeathsData: any;
+  hospitalBedsPerDate: any;
   activeCases: number;
-  numberOfCases: number;
   deaths: number;
   comparison: any;
-  province: any;
   relative = false;
   options: any;
+  covidOverview: CovidOverview;
 
 
   constructor(private covidService: CovidService, private socketService: SocketService) {
 
   }
 
   public ngOnInit(): void {
+
+    this.initializeAll();
+
+    this.socketService.connectAndObserveNewData()
+      .subscribe(() => {
+        this.initializeAll();
+      });
+  }
+
+  private initializeAll(): void {
     this.initializeComparisonCasesChart(this.relative);
     this.initializePositiveCasesPerDateChart();
```

Comment:
Please adapt the labels of all of the charts (and also the table) which have date as labels. It is showing date and hh:mm:ss. Change it to date only